docs/api/websocket.md

```diff
@@ -525,3 +525,28 @@ If an error occurs, the `success` key in the `result` message will be set to `fa
    }
 }
 ```
+
+### Error handling during service calls and translations
+
+When a `ServiceValidationError` (`service_validation_error`) exception was raised during a websocket service call. A stack trace is printed to the logs at debug level only. Raising `HomeAssistantError` (`home_assistant_error`) or subclass exception will still print a full stack trace to the logs.
+
+During a websocket service calls translations are supported for `HomeAssistantError` and its subclasses. When a `translation_key` is set by the caller of the exception this, the keys `translation_domain` and `translation_placeholders` will be added to the error response.
```

Comment:
I'd just say that translations are supported for `HomeAssistantError` etc, and then describe the response items. Who is setting what and how doesn't matter. This page documents the websocket API, not how exceptions are raised.